Dockerfile.j2

```diff
@@ -28,10 +32,9 @@ RUN yum install -y yum-utils \
     && gpg --batch --verify {{ CRATE_TAR_GZ }}.asc {{ CRATE_TAR_GZ }} \
     && rm -rf "$GNUPGHOME" {{ CRATE_TAR_GZ }}.asc \
     && tar -xf {{ CRATE_TAR_GZ }} -C /crate --strip-components=1 \
-    && rm {{ CRATE_TAR_GZ }} \
-    && ln -sf /usr/bin/python3.6 /usr/bin/python3
```

Comment:
Did you check if crash still works without it?

Reply:
Hi Mathias,

thanks for your attention. I just added 185c15853 which outlines the process to verify that `crash` still works as intended.

```
# Build
export CRATEDB_VERSION=4.8.0
python3 update.py --cratedb-version ${CRATEDB_VERSION} > Dockerfile.probe
docker build --file Dockerfile.probe --tag local/crate:${CRATEDB_VERSION} .

# Test
docker run -it --rm local/crate:${CRATEDB_VERSION} crash --version
0.27.0
```

With kind regards,
Andreas.

Reply:
As the patch is already acknowledged, I assume it will be good to go now and I will merge it. Let me know if you have further suggestions. Maybe @seut also wants to have a final glimpse.